cannon/mipsevm/testutil/mips.go

```diff
@@ -182,11 +182,11 @@ func ValidateEVM(t *testing.T, stepWitness *mipsevm.StepWitness, step uint64, go
 
 // AssertEVMReverts runs a single evm step from an FPVM prestate and asserts that the VM panics
 func AssertEVMReverts(t *testing.T, state mipsevm.FPVMState, contracts *ContractMetadata, tracer *tracing.Hooks) {
-	insnProof := state.GetMemory().MerkleProof(state.GetPC())
```

Comment:
I'd suggest accepting an expected revert message here.  Then you can check it with:
```
    retVal, _, err := env.Call(vm.AccountRef(sender), contracts.Addresses.MIPS, input, startingGas, common.U2560)
	require.Contains(t, string(retVal), expectedRevertMsg)
```